openapi/listings/listings.yaml

```diff
@@ -447,3 +570,307 @@ components:
+                closeTime: '09:45'
+      description: |-
+        The operating hours for the listing profile. 
+
+        Each entry in the array is a set of hours for a particular department or customer of the business. It is recommended to provide the `general` hours for all locations. Some locations may have additional definitions for 
+      items:
+        type: object
+        properties:
+          hoursTypeId:
+            type: string
+            description: |-
+              The category of hours that may apply to a specific department or service. Not all types are valid for all business categories.
+
+              The general hours for a location must be set before defining hours for other departments. 
+
+              Currently only `general` is supported in the Vendasta platform. We intend to add others in the future.
+            enum:
+              - general
+              - delivery
+              - kitchen
+              - pickup
+              - driveThrough
+              - takeout
+          regularHours:
+            type: array
+            description: |-
+              A collection of times that this location is open for business. Each period represents a range of hours when the location is open during the week.
+
+              You may specify multiple entries for the same date if you are open for multiple periods (Example lunch and supper).
+            items:
+              type: object
+              properties:
+                openDay:
+                  allOf:
+                    - type: string
+                      x-stoplight:
+                        id: 3u07tkjf0bxdg
+                      description: Indicates the day of the week that this period starts on.
+                    - $ref: '#/components/schemas/dayOfTheWeek'
+                closeDay:
+                  allOf:
+                    - type: string
+                      x-stoplight:
+                        id: svnvxunjr767f
+                      description: Indicates the day of the week that this period ends on. It should be the same as `openDay` except when the period goes over midnight. The total period may not be more than 24 hours
+                    - $ref: '#/components/schemas/dayOfTheWeek'
+                openTime:
+                  $ref: '#/components/schemas/timeOfDay'
+                closeTime:
+                  $ref: '#/components/schemas/timeOfDay'
+              required:
+                - openDay
+                - closeDay
+                - openTime
+                - closeTime
+          specialHours:
+            type: array
+            description: |-
+              `specialHours` represents a set of time periods when a location's operational hours differ from its regular business hours. This may be a holiday or special event.
+              These hours replace all regular hours for the day. The `openTime` and `startDate` must predate the `closeTime` and `endDate` respectively.
```

Comment:
I think this is incorrect, they should only replace the sets of hours that begin on the dates specified by the special hours entry.